apps/web/src/db/schema/prompts/prompts.ts

```diff
@@ -1,9 +1,20 @@
+import { latitudeSchema } from '$/db/schema/schema'
 import { timestamps } from '$/db/schema/schemaHelpers'
 import { InferSelectModel } from 'drizzle-orm'
-import { pgTable } from 'drizzle-orm/pg-core'
+import { serial, text, uniqueIndex, varchar } from 'drizzle-orm/pg-core'
 
-export const prompts = pgTable('prompts', {
-  ...timestamps(),
-})
+export const prompts = latitudeSchema.table(
+  'prompts',
+  {
+    id: serial('serial').notNull().primaryKey(),
+    path: varchar('path').unique().notNull(),
+    content: text('content').notNull(),
+    hash: varchar('hash').notNull(),
+    ...timestamps(),
+  },
+  (promp) => ({
+    path: uniqueIndex('prompt_path_idx').on(promp.path),
+  }),
+)
```

Comment:
Is this supposed to be the "snapshot"?

Reply:
Also, as previously mentioned, it's missing a `promptId`

Reply:
Yes, this is the snapshot

Reply:
maybe we have to call it prompt_snapshot and call prompt_versions prompts instead

OPINIONS EVERYWHERE

Reply:
I think we should have `prompt_versions` and `prompt_snapshots`. `prompts` is just something built from those, something theoretical and untangible ✨